README.md

```diff
@@ -1,84 +1,50 @@
 # Mashery Toolbelt
 CLI tool for Mashery API Management provisioning.
 
-## Functionality
 
-```
-create-errorset    Create probem+json Mashery errorset for an API based on adidas API Guidelines.
-ls                 List existing API services and their ids.
-config             Prints config information.
-```
+## Functionality
 
-## Installation
-With Node.js v7.5.0 or higher installed, run
+- Provide various commands to run complex scenarios in Mashery
+- App data are stored in `HOMEDIR/.mashery-toolbelt/*`
```

Comment:
I'd like to have here more clarification on where it is stored on Windows machines. Maybe Nitesh can help? Non blocking for the PR tho.